src/main/resources/templates/MonitoredTripSms.ftl

```diff
@@ -0,0 +1,13 @@
+<#--
+    This is a template for an SMS text that gets sent when there are new notifications about an
+    OTP user's monitored trip.
+-->
+
+Your trip has the following notifications:
+<#list notifications as notification>
+  -${notification}
+</#list>
+
+View trip: ${OTP_UI_URL}/#/savedtrips/${tripId}
+
+To stop receiving notifications, reply STOP.
```

Comment:
Unless we are actively working on listening to replies to these SMS, we shouldn't mention this.

Reply:
@binh-dam-ibigroup, I think this generally comes out-of-the-box with any text messaging service: https://support.twilio.com/hc/en-us/articles/223134027-Twilio-support-for-opt-out-keywords-SMS-STOP-filtering-